src/awscli_login/__init__.py

```diff
@@ -81,6 +81,11 @@ class Login(BasicCommand):
             'default': False,
             'help_text': 'Forces a login attempt to the IdP using cookies'
         },
+        {
+            'name': 'duration',
+            'default': None,
+            'help_text': 'time in seconds that the token will last (disables daemon/refresh)'
```

Comment:
Please add a `disable-refresh` option for disabling the refresh code.

Reply:
The `duration` option should only be used to set the `duration` not be used to disable the refresh daemon.